src/main.cpp

```diff
@@ -243,6 +242,7 @@ void npl_freertos_hw_set_isr(int irqn, void (*addr)(void)) {
     case RTC0_IRQn:
       rtc0_isr_addr = addr;
       break;
+    default:;
```

Comment:
I don't think it's an issue and I'm not sure I want to change the behaviour in this PR. Also debug builds are broken currently and I don't know if anyone's even using them.

Reply:
Maybe just add a `break` since each switch case should contain it.

From The C programming language - Second edition (K&R 2):
> Chapter 3.4 Switch
> 
>    As a matter of good form, put a break after the last case (the default here) even though it's logically unnecessary. Some day when another case gets added at the end, this bit of defensive programming will save you.